src/config.py

```diff
@@ -56,6 +66,8 @@ def port(self) -> int:
             return SECURITY_PROTOCOL_PORTS[self.protocol].internal
         elif self.scope == "EXTERNAL":
             return SECURITY_PROTOCOL_PORTS[self.protocol].external
+        else:
+            raise ValueError("Only EXTERNAL and INTERNAL scopes are accepted")
```

Comment:
Scope is validated on the setter. That happens during init, so this is not needed.

Reply:
I know, the issue is with the type-hint returning `int`. We have two `if`s, else `None`.
So in 'theory' it could return `None`, hence the check adding.

Reply:
![image](https://user-images.githubusercontent.com/51744472/212660907-e1d6e194-d550-4194-9beb-d462e4b4be4e.png)

Reply:
Then remove the `elif` and do a return directly, so the type hinting doesn't complain.

Reply:
Got it, I'll default to `internal` then for safety.